### PR TITLE
MB-4432 Desktop Container Styling Applied to Customer Onboarding

### PR DESCRIPTION
**Pull request by Shkeating**
## Description
This branch establishes the **SectionWrapper** component, a wrapping div with css applied to establish white boxes and spacing changes to sections in the customer app on desktop layouts. This branch also sets the background color of the customer app to be the **$bg-gray** color established in the MilMove Design System (MMDS). We also tweak backgound colors and some spacing for some components in the system that became apparent after the change in background color. For the end user, this change will break up complex page layouts into easier to read sections, and will help them know which elements are interactive on the page.

## Reviewer Notes

I tried my best to ensure that I implemented <SectionWrapper> on all the pages where it is needed but if I missed something let me know!

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
yarn install
yarn storybook
```

OR 

```sh
make client_run
make server_run
```

## Code Review Verification Steps
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-4432?atlOrigin=eyJpIjoiOWFlYmUwZGVhYmUyNGYyN2I0NTQ1ODBhYmFkMmNiNWMiLCJwIjoiaiJ9) for this change
* [Abstract spec](https://share.goabstract.com/376a7912-b8dd-4633-bcd0-7dfb2fc07735?mode=build&sha=8a7a7045845cb1fdf125dbeaa17142fa27afcf87) 

## Screenshots

HHG Details form
Home Page
Review Page
Shipment Detail Review

### CODE REVIEW

**Review by amanda5iu — APPROVED** [review]
Beautiful work, Shauna!

**Comment by Shkeating** [comment]
still need to determine what we want to do with the following pages:

- legalese/signature page
- How much do you think you'll move? page with the slider
- page listing move types in h3s (need to go back through and find the name)

still need to move `.wrapper-co` styling out of global style sheet

**Review by kimallen — COMMENTED** [review]
This is great!  And that's a lot of files!

I noticed the edit pages don't utilize the SectionWrapper, not sure if we intend to at this point since those pages may get an overhaul later. 

 This includes `EditProfile`, `EditBackupContact`, `EditContactInfo`, `EditOrders`, `EditWeight`.   (You can't navigate to EditBackupContact and EditContactInfo right now, but they are found at `/moves/review/edit-backup-contact` and `/moves/review/edit-contact-info`. Same with EditWeight? `/moves/:moveId/review/edit-weight`)

**Comment by suzubara** [comment]
IMO let's handle the edit pages later on? This PR contains a lot of changes already and it would be good for them to be merged in.

File: src/components/Customer/Review/ShipmentCard/PPMShipmentCard/index.jsx [hunk 1]
```diff
@@ -68,8 +68,8 @@ const PPMShipmentCard = ({
             <dd>{estimatedWeight} lbs</dd>
           </div>
           <div className={`${styles.row} todo`}>
-            <dt>Estimated incentive</dt>
-            <dd>{estimatedIncentive || 'Rate info unavailable'}</dd>
+            <dt className={styles.noborder}>Estimated incentive</dt>
+            <dd className={styles.noborder}>{estimatedIncentive || 'Rate info unavailable'}</dd>
```
Comment (suzubara) [inline src/components/Customer/Review/ShipmentCard/PPMShipmentCard/index.jsx:72]: What is this class for? Can we use a pseudo selector instead?

File: src/components/Customer/SelectableCard.jsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { string, func, bool, node } from 'prop-types';
 import { Radio, Button } from '@trussworks/react-uswds';
 import classnames from 'classnames';
 import FontAwesomeIcon from '@fortawesome/react-fontawesome';
-import faQuestionCircle from '@fortawesome/fontawesome-free-regular/faQuestionCircle';
+import faQuestionCircle from '@fortawesome/fontawesome-free-regular';
```
Comment (suzubara) [inline src/components/Customer/SelectableCard.jsx:6]: This should not be changed

File: src/components/Table/Table.jsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { useTable } from 'react-table';
+import { useTable } from 'react-table/index';
```
Comment (suzubara) [inline src/components/Table/Table.jsx:2]: This should also not be changed

**Review by suzubara — CHANGES_REQUESTED** [review]

File: src/components/Customer/Review/ShipmentCard/PPMShipmentCard/index.jsx (hunk 1 above)
Comment (Shkeating) [reply src/components/Customer/Review/ShipmentCard/PPMShipmentCard/index.jsx:72]: not without affecting all the other shipment types, which we don't want

Comment (suzubara) [reply src/components/Customer/Review/ShipmentCard/PPMShipmentCard/index.jsx:72]: what is different about this shipment type?

**Review by suzubara — APPROVED** [review]

**Pull request merged**